frontend/src/components/calendar/CalendarCard/CalendarCard.tsx

```diff
@@ -51,7 +51,7 @@ const CalendarCard = ({
   }, []);
 
   return (
-    <li className={`position-relative`}>
+    <li className={`position-relative nav-calendar-card`}>
```

Comment:
also switch this, isn't ir more logical to put nav calendar card first and then postition-relative?
or maybe just put the postiion relative into the nav-calendar-card in the scss file